content/post/105-deprecating-secio.md

```diff
@@ -0,0 +1,51 @@
+---
+date: 2020-08-05
+url: /2020-08-05-deprecating-secio/
+title: We're removing support for the SECIO security transport
+author: Jacob Heun
+tags: ipfs, libp2p, secio
+---
+
+## A brief history
+
+SECIO is a TLS like security transport that was developed for IPFS and libp2p in 2014 to alleviate a gap with TLS1.2 requiring Certificate Authorities. SECIO has been the primary security transport for all libp2p implementations as it's underlying alrogithms are widely supported. In May 2019, go-ipfs 0.4.21 added support for TLS1.3. In go-ipfs 0.5 it became the default security transport. However, it's still not the default security transport for the various libp2p implementations because support is still limited, and we can't leverage it in browsers. While web browsers have introduced support for TLS1.3, we have no way of attaching the necessary identity information for libp2p.
+
+So, how is SECIO going away if we can't support TLS1.3 everywhere? The answer is the [Noise security transport](noise_spec), which was created leveraging the [Noise Protocol Framework](noise). Noise provides a set of patterns for composing widely supported cryptographic primitives, which allows us to more easily add support across the various libp2p implementations. Support was added for Noise in go-ipfs 0.6 and js-ipfs 0.47, including rollouts to IPFS infrastructure that Protocol Labs runs.
```

Comment:
```suggestion
So, how can we successfully deprecate SECIO and move on to more modern and widely-used security transports if we can't support TLS1.3 everywhere? The answer is the [Noise security transport](noise_spec), which was created leveraging the [Noise Protocol Framework](noise). Noise provides a set of patterns for composing widely supported cryptographic primitives, which allows us to more easily add support across the various libp2p implementations. Support was added for Noise in go-ipfs 0.6 and js-ipfs 0.47, including rollouts to IPFS infrastructure that Protocol Labs runs.
```

Reply:
For the final clause (including rollouts....) - can we make that less PL focused and more about "and has been updated across major infra in the IPFS network"?